tests/net/ieee802154/fragment/src/main.c

```diff
@@ -29,7 +29,7 @@
 #define NET_LOG_ENABLED 1
 #include "net_private.h"
 
-#define DEBUG 0
+#define DEBUG 1
```

Comment:
was this intentional?

Reply:
It was for testing the patch, but that should not be here indeed.